website/content/commands/acl/translate-rules.mdx

```diff
@@ -10,6 +10,8 @@ It will be removed in a future major release when support for the legacy ACL sys
 
 Command: `consul acl translate-rules`
 
```

Comment:
@markan : good catch. I recommend handling that as a separate PR. We can wait to merge it until "versioned docs" is rolled out at end of week so that it just doesn't exist in Consul 1.11+ docs pages. (Actually, I'll check with the docs team how we want to handle removing functionality once we have versioned docs.)